frontend/lib/src/declarations.d.ts

```diff
@@ -27,3 +27,11 @@ declare module "fzy.js" {
   export function positions(pattern: string, subject: string): Array<number>
   export function hasMatch(pattern: string, subject: string): boolean
 }
+
+declare module "react-color/es/components/common/Saturation" {
```

Comment:
Had to add a type definition for Saturation here (which is internal in react-color), otherwise TypeScript complained about the security error workaround. Lmk if there's a better way to fix this!

Reply:
Can you add a comment here in the code to explain why it needs to be added

Reply:
Done